examples/docs_snippets/docs_snippets/concepts/partitions_schedules_sensors/sensors/sensors.py

```diff
@@ -246,7 +246,7 @@ def send_slack_alert():
 # start_cross_repo_run_status_sensor
 
 
-@run_status_sensor(
+@run_status_sensor(  # type: ignore[arg-type]
```

Comment:
I think users are likely to hit this too

I think the preferred fix would be to change the type annotation for `@run_status_sensor` from `Callable[[RunStatusSensorContext], RawSensorEvaluationFunctionReturn]` to `Callable[[...], RawSensorEvaluationFunctionReturn]`

(to reflect that fact that we don't actually require a context object)